eng/Subsets.props

```diff
@@ -71,8 +71,7 @@
     <DefaultLibrariesSubsets Condition="'$(BuildTargetFramework)' == '$(NetCoreAppCurrent)' or
                                         '$(BuildTargetFramework)' == '' or
                                         '$(BuildAllConfigurations)' == 'true'">libs.native+</DefaultLibrariesSubsets>
-    <DefaultLibrariesSubsets>$(DefaultLibrariesSubsets)libs.sfx+libs.oob</DefaultLibrariesSubsets>
-    <DefaultLibrariesSubsets Condition="'$(DotNetBuildFromSource)' != 'true'">$(DefaultLibrariesSubsets)+libs.pretest</DefaultLibrariesSubsets>
+    <DefaultLibrariesSubsets>$(DefaultLibrariesSubsets)libs.sfx+libs.oob+libs.pretest</DefaultLibrariesSubsets>
```

Comment:
This is necessary because the RuntimeList.xml file is only created when running the libs.pretest subset.

Reply:
Why was the DotNetBuildFromSource condition added initially? Is removing it going to break source build?

Reply:
`DotNetBuildFromSource` skips this because it doesn't need a layout as it doesn't run tests. Shouldn't we get the xml manifests from the installer partition instead of relying on the test layouts?